test/services/SubscriptionServiceSpec.scala

```diff
@@ -165,9 +165,9 @@ class SubscriptionServiceSpec extends SpecBase with MockServiceApp with MockitoS
           .set(SecondContactPage, false)
           .success
           .value
-          .set(RegistrationInfoPage, RegistrationInfo.build("safeId", AsIndividual))
-          .success
-          .value
+//          .set(RegistrationInfoPage, RegistrationInfo.build("safeId", AsIndividual))
+//          .success
+//          .value
 
         val response: EitherT[Future, ApiError, SubscriptionID] = EitherT.fromEither[Future](Left(error))
         when(mockSubscriptionConnector.readSubscription(any())(any(), any())).thenReturn(Future.successful(None))
```

Comment:
Same here if any reason to keep commented code

Reply:
Removed